collector/exporters/mockdestinationexporter/exporter.go

```diff
@@ -0,0 +1,63 @@
+package mockdestinationexporter
+
+import (
+	"context"
+	"errors"
+	"math/rand/v2"
+	"time"
+
+	"go.opentelemetry.io/collector/exporter"
+
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	"go.uber.org/zap"
+)
+
+type MockDestinationExporter struct {
+	config *Config
+	logger *zap.Logger
+}
+
+func NewMockDestinationExporter(config *Config,
+	params exporter.Settings) (*MockDestinationExporter, error) {
+
```

Comment:
nit: maybe worth to add small config validation ?
ResponseDuration > 0 
1 > RejectFraction > 0

Reply:
added in config.go under `Validate` function